@kiva/kv-components/vue/KvToast.vue

```diff
@@ -71,6 +71,10 @@
 							tw-flex
 						"
 						>
+							<slot
+								v-if="hasParentContentSlot"
+								name="parentContent"
```

Comment:
Calling this slot something like `toastContent` might make more sense.

Reply:
done